pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -284,10 +284,22 @@ func validateStateChangeRequests(ar *v1beta1.AdmissionRequest, vm *v1.VirtualMac
 		}
 
 		if getRenameRequest(existingVM) != nil {
-			return []metav1.StatusCause{{
-				Type:    metav1.CauseTypeFieldValueInvalid,
-				Message: "Modifying a VM during a rename process is not allowed",
-			}}
+			allowed := webhooks.GetAllowedServiceAccounts()
+			if _, ok := allowed[ar.UserInfo.Username]; ok {
```

Comment:
shouldn't `ok` -> `!ok`

Reply:
@xpivarc If it's ok then the user is a service account. For a service account we allow to change the metadata and status areas. Even a service account cannot change the spec during rename process.